release/rancher/rancher.go

```diff
@@ -326,19 +316,10 @@ func CreateRelease(ctx context.Context, ghClient *github.Client, r *ecmConfig.Ra
 	opts.Draft = !preRelease
 	opts.ReleaseNotes = ""
 
-	fmt.Printf("creating release with options: %+v\n", opts)
-	if r.DryRun {
-		fmt.Println("dry run, skipping tag creation")
-		return nil
-	}
-
 	createdRelease, err := repository.CreateRelease(ctx, ghClient, opts)
-	if err != nil {
-		return err
-	}
-	fmt.Println("release created: " + *createdRelease.URL)
 
-	return nil
+	// GetHTMLURL will return an empty value if it isn't present
```

Comment:
Why remove the error check on the CreateRelease call?

Reply:
Because `createdRelease.GetHTMLURL()` returns an empty string if it isn't available, returning it and `err` is the same as checking `err` and returning an empty string

```go
createdRelease, err := repository.CreateRelease(ctx, ghClient, opts)
if err != nil {
    return "", err
}
return createdRelease.GetHTMLURL(), nil
```
Is the same as
```go
createdRelease, err := repository.CreateRelease(ctx, ghClient, opts)
return createdRelease.GetHTMLURL(), err
```